markdown/articles/2021/12/24/mindkiller-update.md

```diff
@@ -0,0 +1,345 @@
+---
+authors: d8dk32
+banner: SotV
+title: "50-Match Kumite: Mindkiller Kata"
+---
+
+I've once again been brewing around [Mindkiller
+Doomsday][articles:the-mindkiller], sometimes known as All Bad Cards Doomsday.
+I'd made some changes since my last update on the deck and it felt quite strong
+in testing, so I felt it was time to write another update. This time, I wanted
+to collect some data to help back up my claim that this deck can be competitive
+in the current legacy metagame. So I resolved to play through 10 leagues and
+track my performance. Fifty matches, no surrender, no dropping from leagues. I
+focused my data tracking on match wins, so there's no per-game data, but I did
+track by opposing archetype to better understand how the deck fares against
+different apects of the metagame. It should also be noted that 50 matches isn't
+much in the grand scheme of things, but it should give us somethng to talk
+about.
+
+[articles:the-mindkiller]: /articles/2020/03/21/the-mindkiller/
```

Comment:
```suggestion
[articles:the-mindkiller]: /articles/2020/03/21/the-mindkiller
```

AFAIK I don't take care of duplicate URLs (I should) so linking paths with and without trailing `/` *hurts* SEO for when we starting caring about it

Reply:
fixed